tools/test/run_pylint.sh

```diff
@@ -1,19 +1,13 @@
 #!/bin/bash -u
 root=`dirname $0`
 num_files_checked=0
-num_errors_found=0
-dirs=( $@ )
 declare -a files_to_check=()
 
 echo "INFO: starting pylint tests"
-for dir in ${dirs[@]}; do
-    for f in `find $dir -type f| grep -v fake_data_root`; do
-        if `file $f| grep -q "Python script"`; then
-            files_to_check+=( $f )
-            num_files_checked=$((num_files_checked + 1))
-        fi
-    done
-done
+files_to_check=($(git ls-files "*.py"))
```

Comment:
```suggestion
files_to_check=($(git ls-files '*.py'))
```
to prevent shell expansion.

Reply:
Glob expansion doesn't occur inside double quotes, either.

Reply:
Ah you are right. So either `'` or `"` would work.

Reply:
Done